image/tarexport/tarexport.go

```diff
@@ -30,18 +32,36 @@ type tarexporter struct {
 	loggerImgEvent LogImageEvent
 }
 
+type ocitarexporter struct {
+	te   *tarexporter
+	name string
```

Comment:
I don't know - the same it's already having with the normal one https://github.com/docker/docker/blob/master/image/tarexport/load.go#L26 ?